.travis.yml

```diff
@@ -1,3 +1,43 @@
+sudo: required
+dist: trusty
+
 language: node_js
 node_js:
-  - "stable"
\ No newline at end of file
+  - 6
+
+env:
+  -TZ="America/New_York"
+# Check versioning immediately before doing anything else
+before_install:
+  - echo 'EXISTING TAGS' && git tag
+  - 'export VERSION=v$(node -p "require(\"./package.json\").version")'
+  - echo CURRENT VERSION ${VERSION}
+  - "if [ -z ${TRAVIS_TAG} ]; then if [ -z `git tag | awk -v env_var=${VERSION}  -F: '$1==env_var {print $1}'` ]; then echo 'Build is a new tag';else echo 'BUILD IS AN OLD TAG! UPDATE PACKAGE VERSION' && exit 1;fi;else if [ $TRAVIS_TAG = $VERSION ]; then echo 'Tags match'; else echo 'ERROR TAG VERSION DOES NOT MATCH PACKAGE VERSION' && exit 1; fi;fi;" 
+  - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
+  - npm i -g npm@5.3.0
+
+# Check linting and build
+before_script:
+  - npm run lint
+
+after_success:
+  - babel ./src --out-dir ./lib && gulp dist
```

Comment:
Looks like this command is the same as the `prepublish` script in the package.json! Should we reference that command here? Or even better- since `prepublish` is now deprecated, update the command to be called `build`?

Reply:
Works for me!